### ISSUE TITLE
Strange Styling Applied

### ISSUE THREAD

**Issue by andreencar** [shoutem/ui#125]
Hi, I've just incorporated ShoutemUI onto my react native app and found that the buttons on a navigationBar appear not centralized with the text on the middle and that the Button appears like it is not styled, and an image within a ScrollView is set as if it was float:left instead of centered aswell.

I'll try to upload a photo later on, but do I need to import something more for all the styling to work rather than what I need from the "@shoutem/ui"? Maybe the fact that they are being instanced by the NavigatorIOS react native component is changing the styles?

I'm really motivated to use this ui library and would not like to dismiss it for probably simple things.

P.S. I've had an issue with the navigation bar that was solved on an issue by passing the stylename 'inline'. I had to delve into github and issues to identify the solution, when it could be as simple as looking into the documentation if that styling was defined there. When is the documentation going to be updated?

**Comment by felagund18** [comment]
Its also strange styling
![screen shot 2017-01-08 at 1 04 58 pm](https://cloud.githubusercontent.com/assets/1384327/21747486/1f1b0726-d5a3-11e6-8cc9-c07826c7ddf6.png)

**Comment by sky-andreencarnacao** [comment]
That's exactly it

**Comment by cmeiller** [comment]
Same with <Text> in <Button>, any idea?
<img width="319" alt="capture d ecran 2017-01-12 a 12 59 01" src="https://cloud.githubusercontent.com/assets/412839/21888976/ee37165a-d8c6-11e6-9c23-c0ba33af1bb7.png">

**Comment by SidBala** [comment]
Hitting same thing here.

**Comment by SoHotSoup** [comment]
Hey guys! We have two `NavigationBar` components available, one to import directly from `@shoutem/ui` and other to be used as child component of our `CardStack` available in `@shoutem/ui/navigation`. Its example usage is shown in RestaurantsApp of our examples. Could you inform me what of those two components you are using?
Also, we are aware of such alignment issue on Android with one from `@shoutem/ui/navigation` and I hope we will fix it this or next week. But last image by @cmeiller looks like it has been taken on iOS, and its really strange that left and right Button are unaligned. @cmeiller could you please share your code with me? Maybe I have a clue why that is happening to you.

**Comment by cmeiller** [comment]
@SoHotSoup thanks for your answer.
Indeed the screen is from iOS. I use `react-native-router-flux` and extend their NavBar with yours like that:
```js
import React from 'react';
import { NavBar } from 'react-native-router-flux';
import { Button, Icon, NavigationBar, Text, Title } from '@shoutem/ui';

class MainNavBar extends NavBar {
  ...

  render() {
    return (
      <NavigationBar
        styleName="clear"
        leftComponent={<Button><Text>Test</Text></Button>}
        centerComponent={<Title>{this.props.title}</Title>}
        rightComponent={<Icon name="ic_user_profile" onPress={...} />}
      />
    );
  }
}
```

Title and Icon are aligned correctly but not the `Test` text.

Is it a problem because I didn't used the second `NavigationBar` from `@shoutem/ui/navigation` ?
I missed it.

**Comment by keyeh** [comment]
The entire `NavigationBar` seems to be spaced incorrectly for me. Using [navbar + icon example](http://shoutem.github.io/docs/ui-toolkit/components/navigation-bar#navbar--icon) from the doc


![image](https://cloud.githubusercontent.com/assets/5600824/22771608/9df6785c-ee4c-11e6-963c-6a9e0f175054.png)

**Comment by sisiguo** [comment]
Any updates on this issue? Running into the same problem using the `@shoutem/ui` `NavigationBar`...

**Comment by latte-diygame** [comment]
Me too. I want to start react native with `@shoutem/ui` but same problem..

**Comment by zrumenjak** [comment]
This was fixed in #216
